mttl/utils.py

```diff
@@ -555,3 +198,18 @@ def wrapped_fn(*args, **kwargs):
 
 def generate_random_string(str_len=10):
     return "".join(random.choices(string.ascii_uppercase, k=str_len))
+
+
+# from projects.modular_llm.src.utils.evaluators import Evaluator
+def get_loss(model, evaluator: "Evaluator", **kwargs):
```

Comment:
same , maybe move somewhere else? in evaluator?

Reply:
actually, not used anywhere